common/app/model/meta.scala

```diff
@@ -23,6 +23,7 @@ import play.api.libs.functional.syntax._
 import play.api.mvc.RequestHeader
 import navigation.GuardianFoundationHelper
 import services.newsletters.{NewsletterResponse}
+import org.joda.time.DateTimeZone
```

Comment:
I know that @shtukas is working quite hard to remove `joda.time` from frontend, if this code is going to survive the test - it might be worth switching it over to the replacement library (I think one of the Java built-ins?)

cc @shtukas might have more info on a replacement function for this use-case

Reply:
Thanks @OllysCoding . For the moment it's ok. I will made the switch later when I go through this code :)

Reply:
That's letting that through. I'll migrate it in the next PR.